src/aat/resources/features/F-102/S-539.td.json

```diff
@@ -0,0 +1,45 @@
+{
+  "_guid_": "S-539",
+  "_extends_": "F-102_Test_Data_Base",
+  "title": "must return 404 for if no jurisdiction is found",
+  "specs": [
+    "a detailed profile in CCD having CR case access for a jurisdiction",
+    "returns the error message : No jurisdictions found for given access criteria",
+    "has UPDATE as case access parameter"
+  ],
+  "productName": "CCD Data Store",
+  "operationName": "Get jurisdictions available to the user",
+  "user": {
+    "username": "befta.caseworker.2@gmail.com",
+    "password": "Pa55word11"
+  },
+  "request": {
+    "pathVariables": {
+      "uid": "[[DYNAMIC]]"
+    },
+    "queryParams": {
+      "access": "create"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_400_Response",
+    "headers": {
+      "Content-Length": "[[ANY]]",
+      "Transfer-Encoding": "[[ANY]]",
+      "Connection": "[[ANY]]",
+      "Content-Type": "[[ANY]]",
+      "Content-Encoding": "[[ANY]]"
+    },
+    "body": {
+      "exception": "uk.gov.hmcts.ccd.endpoint.exceptions.BadRequestException",
+      "timestamp": "[[ANY]]",
+      "status": 400,
+      "error": "Bad Request",
```

Comment:
This is also inherited from common_400_response

Reply:
Fixed